content/docs/04-clusters/02-data-center/03-vmware.md

```diff
@@ -76,6 +76,21 @@ The following prerequisites must be met before deploying a Kubernetes clusters i
 11. Zone Tagging: A dynamic storage allocation for persistent storage.
 
 
+<InfoBox>
+
+### Naming conventions for vSphere Region and Zone Tags
```

Comment:
This feels like prerequisite

Reply:
yes added as an infobox under the prerequisite section